src/components/LuxLibraryLogo.vue

```diff
@@ -69,21 +69,18 @@ export default {
     }
   }
   &.light {
-    &:hover,
     &:focus-within {
       @include princeton-focus(light);
       outline-offset: 0.1rem;
     }
   }
   &.dark {
-    &:hover,
     &:focus-within {
       @include princeton-focus(dark);
       outline-offset: 0.1rem;
     }
   }
   &.shade {
-    &:hover,
```

Comment:
On hover in shade theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-black);
}